src/portalbackend/PortalBackend.PortalEntities/Entities/Agreement.cs

```diff
@@ -62,6 +62,7 @@ public Agreement(Guid id, AgreementCategoryId agreementCategoryId, string name,
     public Guid? UseCaseId { get; set; }
 
     public Guid? DocumentId { get; set; }
+    public string? Agreement_link { get; set; }
```

Comment:
Please rename the property and adjust the usages, the Migration needs to be recreated as well, since the PortalDbContextModelSnapshot contains the property name

```suggestion
    public string? AgreementLink { get; set; }
```

Reply:
fixed